runtime/CMakeLists.txt

```diff
@@ -108,11 +100,16 @@ ExternalProject_Add(BuildKLEERuntimes
   CONFIGURE_COMMAND "${CMAKE_COMMAND}" -E echo "" # Dummy command
   # `env` is used here to make sure `MAKEFLAGS` of KLEE's build
   # is not propagated into the bitcode build system.
-  BUILD_COMMAND ${ENV_BINARY} MAKEFLAGS="" ${MAKE_BINARY} -f Makefile.cmake.bitcode all
-  ${EXTERNAL_PROJECT_BUILD_ALWAYS_ARG}
+  BUILD_COMMAND "${CMAKE_COMMAND}" -E echo "" # Dummy command
   INSTALL_COMMAND "${CMAKE_COMMAND}" -E echo "" # Dummy command
 )
 
+ExternalProject_Add_Step(BuildKLEERuntimes RuntimeBuild
```

Comment:
It would also be a good idea to have a comment here we are doing it this way to avoid the fact that `BUILD_ALWAYS` is not supported by CMake < 3.1.